fuzzers/int_loop.mk

```diff
@@ -42,7 +42,7 @@ ifneq ($(QUICK),Y)
 	${XRAY_DBFIXUP} --db-root build --clb-int
 	# https://github.com/SymbiFlow/prjxray/issues/399
 	# Clobber existing .db to eliminate potential conflicts
-	if [ -f ${XRAY_DATABASE_DIR}/${XRAY_DATABASE}/segbits_clb*.db ] ; then cp ${XRAY_DATABASE_DIR}/${XRAY_DATABASE}/segbits_clb*.db build/database/${XRAY_DATABASE}; else bash ${XRAY_DIR}/fuzzers/int_create_empty_db.sh; fi
+	if [ -f ${XRAY_DATABASE_DIR}/${XRAY_DATABASE}/segbits*.db ] ; then cp ${XRAY_DATABASE_DIR}/${XRAY_DATABASE}/segbits*.db build/database/${XRAY_DATABASE}; else bash ${XRAY_DIR}/fuzzers/int_create_empty_db.sh; fi
```

Comment:
This if statement is always false :(

https://stackoverflow.com/questions/2937407/test-whether-a-glob-has-any-matches-in-bash

Reply:
I think this should be even more intelligent. We have quite a few db files and we may hit any combination of them existing or not. IMO we should have a list of db files, iterate over it, copy existing and touch unexisting.

Reply:
or simply create empty db (if none present) in settings script and do not care about checks in fuzzers makefiles

Reply:
I have opened followup PR #531